### DISCUSSION TITLE
Preview vs. Development environments

### DISCUSSION THREAD

**Discussion by ejkreboot** [vercel/community#1817] [Help] [answered]
### Summary

I can assign a git branch to the preview environment in my project dashboard, but I do not seem to be able to set a git branch to the development environment. How do I make use of the development environment?

**Comment by swarnava** [comment, marked as answer]
Hi,

Development environment is only applicable if you use `vercel dev` command via CLI. You won't be able to assign any branch on that specific Environment. Learn more here: https://vercel.com/docs/concepts/deployments/configure-a-build#build-and-development-settings

**Comment by rohity123456** [comment]
can we assign a domain to it ?
